api-report/driver-utils.api.md

```diff
@@ -112,7 +112,7 @@ export function combineAppAndProtocolSummary(appSummary: ISummaryTree, protocolS
 export function configurableUrlResolver(resolversList: IUrlResolver[], request: IRequest): Promise<IResolvedUrl | undefined>;
 
 // @public (undocumented)
-export function createGenericNetworkError(errorMessage: string, canRetry: boolean, retryAfterMs?: number, props?: ITelemetryProperties): GenericNetworkError | ThrottlingError;
+export function createGenericNetworkError(errorMessage: string, canRetry: boolean, retryAfterMs?: number, props?: ITelemetryProperties): ThrottlingError | GenericNetworkError;
```

Comment:
I'm not sure why this would have changed. Was wondering if others have seen this flip-flopping? or if we should care? :)

Reply:
I don't do much dev work in this repository, so I'll write my workflow here in case it helps. Post-integration of changes, I ran `npm i` after updating `prettier` deps to `2.3.1`. Then I used `npm run build:fast` repeatedly to iron out all of the build issues from breaking changes in `@fluid-experimental/tree`. Basically all of those were just due to the `EditNode -> ChangeNode` rename. Since `prettier` got updated, I had to re-format @fluid-experimental/tree with `npx lerna run prettier:fix --scope=@fluid-experimental/tree`, which only updated `Common.ts`. After that I only ran commands which shouldn't make any changes to this (e.g. policy-check, layer-check, etc)

Reply:
CI build complains about API changes related to this file. I think I'm just going to revert this change, but this may be a sign there's something weird with API extractor, its setup in this repo, or some of the code related to this API?

Reply:
that seems to have fixed it. Very odd..